pkg/resource/analyzer/config.go

```diff
@@ -192,6 +192,38 @@ func validatePolicyConfig(schema plugin.AnalyzerPolicyConfigSchema, config map[s
 	return errors, nil
 }
 
+// ValidatePolicyPackConfig validates a policy pack configuration against the specified config schema.
+func ValidatePolicyPackConfig(schemaMap map[string]apitype.PolicyConfigSchema,
```

Comment:
I took this from the pulumi-service, so it is identical for the most part. Maybe we can move to a common pkg where we can import for both the cli and service at some point. There is a func above this that also seems to be doing some schema validation....  but only accpeted the type AnalyzerPolicyConfig, so I would need to do some type of conversion to leverage it, which I fought with for a bit, but in the end I just copied what was being done server side, so at least the way we are validating is consistent with what will be done when the pp is enabled.